api/include/vehicles/kia_soul.h

```diff
@@ -464,16 +464,14 @@ typedef struct
  *
  */
 #define THROTTLE_POSITION_TO_VOLTS_LOW( position ) ( (position) *\
-    (THROTTLE_SPOOF_LOW_SIGNAL_VOLTAGE_MAX - THROTTLE_SPOOF_LOW_SIGNAL_VOLTAGE_MIN) +\
-    THROTTLE_SPOOF_LOW_SIGNAL_VOLTAGE_MIN  )
+    (THROTTLE_SPOOF_LOW_SIGNAL_VOLTAGE_MAX )
```

Comment:
Could we leave it the way it was just in case we end up changing those values again later? (Also for consistency with the other spoof signal conversion macros.)

(Same on Line 474)